File: unicardsoftwaredevelopment-androidmobileappfortopupandrefund-c0e3674d02c4/app/src/main/java/au/com/unicard/tafenswwallet/ui/refund/RefundViewModel.java
package au.com.unicard.tafenswwallet.ui.refund;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class RefundViewModel extends ViewModel {
    private MutableLiveData<String> mText;

    public RefundViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is refund fragment!");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: TAFEWallet/app/src/main/java/com/yuan/tafewallet/ui/refund/RefundViewModel.kt
package com.yuan.tafewallet.ui.refund

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class RefundViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is Refund Fragment"
    }
    val text: LiveData<String> = _text
}